cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/inmemory/InMemoryTableDefinition.java

```diff
@@ -32,6 +34,9 @@
 public class InMemoryTableDefinition
   extends AbstractDatasetDefinition<Table, InMemoryTableAdmin> {
 
+  // TODO: get injection to work
```

Comment:
Yeah, injections weren't setup correctly in our tests for this to work. Fixing it.